apps/prairielearn/elements/pl-dropdown/pl-dropdown.py

```diff
@@ -95,8 +95,8 @@ def render(element_html, data):
                 correct = True
             else:
                 correct = False
-        except Exception:
-            raise ValueError("invalid score" + score)
+        except Exception as err:
+            raise ValueError("invalid score" + score) from err
```

Comment:
Nit, only calling out because this line changed: this is missing a space (any maybe a colon) between the message and the score.